### PR TITLE
feat: add menu font customization

### PR DESCRIPTION
**Pull request by LofiSu**
[[中文版模板 / Chinese template](https://github.com/VisActor/VTable/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a feat: add menu font customization

- [ √] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Chore
- [ ] Release
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/VisActor/VTable/issues/1623

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vtable/examples/interactive/initializeProgressTable.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import * as VTable from '../../src/index.ts';
+import { bindDebugTool } from '../../src/scenegraph/debug-tool/index.ts';
+import { TABLE_EVENT_TYPE } from '../../src/core/TABLE_EVENT_TYPE.ts';
+import { importStyle } from '../../src/components/menu/dom/logic/MenuElementStyle.ts';
```
Comment (Rui-Sun) [inline packages/vtable/examples/interactive/initializeProgressTable.ts:4]: 不建议使用这种调用，如果需要向外暴露，最好都集中在index中

File: packages/vtable/src/components/menu/dom/logic/MenuElementStyle.ts [hunk 2]
```diff
@@ -1,99 +1,104 @@
 import { Env } from '../../../../tools/env';
 
-export function importStyle() {
+interface StyleOptions {
+  fontFamily?: string;
+  fontSize?: string;
+}
+
+export function importStyle(options?: StyleOptions) {
```
Comment (Rui-Sun) [inline packages/vtable/src/components/menu/dom/logic/MenuElementStyle.ts:8]: 样式修改不建议这样使用，全局配置会导致相同项目不同样式互相覆盖；样式相关的配置最好都收敛到theme里，建议在创建相关dom元素时，单独指定style

**Review by Rui-Sun — COMMENTED** [review]
建议修改实现方法

File: packages/vtable/examples/interactive/initializeProgressTable.ts (hunk 1 above)
Comment (LofiSu) [reply packages/vtable/examples/interactive/initializeProgressTable.ts:4]: 收到，马上修改

**Pull request closed without merging**